.travis.yml

```diff
@@ -28,27 +29,14 @@ matrix:
     - pip install -e. -c tests/requirements.txt
     script:
     - make spelling
-  - python: "pypy3.5-5.8.0"
-    env: PYPY_RELEASE="pypy3-v6.0.0-linux64"
   - python: "3.6"
     env: UVLOOP="y"
   allow_failures:
   - env: UVLOOP="y"
-  - python: "pypy3.5-5.8.0"
```

Comment:
Removed it from allow-failures.
Should it stay?

Anyway, should get squash-merged.